rust/dns-over-tcp/src/server.rs

```diff
@@ -215,10 +225,11 @@ impl Server {
     }
 }
 
+#[expect(clippy::type_complexity, reason = "We don't care.")]
```

Comment:
😆

Reply:
YMMV but I think this type-signature is perfectly fine. A fallible function that returns an optional value which is a tuple :shrug: